packages/ketcher-core/src/application/editor/modes/SequenceMode.ts

```diff
@@ -0,0 +1,44 @@
+import { CoreEditor } from 'application/editor/internal';
+import { LayoutMode } from 'application/editor/modes';
+import { BaseMode } from 'application/editor/modes/BaseMode';
+import ZoomTool from 'application/editor/tools/Zoom';
+
+export class SequenceMode extends BaseMode {
+  constructor(previousMode?: LayoutMode) {
+    super('sequence-layout-mode', previousMode);
+  }
+
+  public initialize() {
+    const command = super.initialize();
+    const editor = CoreEditor.provideEditorInstance();
+
+    editor.drawingEntitiesManager.applyFlexLayoutMode();
```

Comment:
seems this line of code will be executed twice since it is also included in `reArrangeChains` func.